ansible/roles/baselayout/vars/main.yml

```diff
@@ -40,10 +40,12 @@ packages: {
   ],
 
   centos7_arm64: ['git'], # git2u not available for aarch64 (yet)
-  centos7_x64: ['git2u','centos-release-scl',], # centos-release-scl is required to enable SCLo
+  centos7_x64: ['devtoolset-6-libatomic-devel', 'git2u','centos-release-scl',], # centos-release-scl is required to enable SCLo
```

Comment:
Not critical, just FYI: I _think_ it might be more efficient to run these as comma-separated strings rather than an array of strings. When it's a CSV I think it does them all at once, when they're separate array elements it does them one after the other. I can't be sure but my impression from flattening some of these in the past is that you get a speed-up from batching.

Reply:
I'll consider that as a followup refactor, or next time I touch this, but I don't want to reansible all the machines now, which I'd feel duty bound to do if I made such a change.

Reply:
actually, I misread, its only x64. changed it.